brainscore/submission/evaluation.py

```diff
@@ -213,11 +224,19 @@ def get_benchmark_instance(benchmark_name):
 
 
 def get_reference(bibtex_string):
-    parsed = bibtexparser.loads(bibtex_string)
-    if len(parsed.entries) > 0:
-        entry = list(parsed.entries)[0]
-        ref, create = Reference.get_or_create(url=entry.get('url', ''),
-                                              defaults={'bibtex': bibtex_string, 'author': entry.get('author', ''),
-                                                        'year': entry.get('year', "")})
+    def parse_bib(bibtex_str):
+        bib_parser = bibtex.Parser()
+        entry = bib_parser.parse_string(bibtex_str)
+        entry = entry.entries
+        assert len(entry) == 1
+        entry = entry.values()[0]
+        return entry
+    try:
+        entry = parse_bib(bibtex_string)
+        ref, create = Reference.get_or_create(url= entry.fields['url'],
+                                               defaults={'bibtex': bibtex_string, 'author': entry.persons["author"][0],
```

Comment:
I think this should be `entry.persons["author"][0].last()` (the `.last()` is missing). Otherwise we'll have first + last